apps/nextjs-website/src/components/molecules/CardsGrid/CardsGrid.tsx

```diff
@@ -29,12 +30,13 @@ const CardsGrid = ({
   cardVariant,
   cardSvg,
   cardSize,
+  containerPx,
 }: CardsGridProps) => {
   const { palette } = useTheme();
   const t = useTranslations('shared');
 
   return (
-    <EContainer>
+    <EContainer px={containerPx} containerSx={{ px: containerPx }}>
```

Comment:
I don't see any difference with:
```suggestion
    <EContainer containerSx={{ px: containerPx }}>
```
why did you use both properties?